### PR TITLE
Week 13 React-Native-App - Ebba Bergholtz

### PR DESCRIPTION
**Pull request by ebbabw**
My reflections on how this week's project turned out:

The time management this week wasn't the best, didn't have much time left to complete it.
But I'm happy on how it turned out.

Things I'd like to have clarified or explained in more detail:

Using Lottie and Loading page before showing items (didn't have so much time to check it out)

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+    <NavigationContainer>
+      <Container>
+    <Stack.Navigator>
+      <Stack.Screen name="Trending this Week" component={Restaurants} />
+      <Stack.Screen name="Best place this week" component={Restaurant} />
+    </Stack.Navigator>
```
Comment (JennieDalgren) [inline App.js:24]: nice that you've tried out navigation!

Comment (JennieDalgren) [reply App.js:24]: although, check your indentation here

File: Pages/Restaurant.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React  from 'react'
+import { Image } from 'react-native';
+import styled from 'styled-components/native' 
+
+
+export const Restaurant = ({ route }) => {
+
+    const { item } = route.params
```
Comment (JennieDalgren) [inline Pages/Restaurant.js:8]: indentation should be 2 spaces, not 4

**Comment by puj** [comment]
The project seems to fetch correctly from the API with me.  @ebbabw Can you confirm that it works for you?

**Comment by ebbabw** [comment]
@puj yes it works for me 👍

**Pull request closed without merging**